packages/esm-hiv-app/src/config-schema.ts

```diff
@@ -0,0 +1,165 @@
+      stopReasonUUID: '163513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      restartReasonUUID: '161011AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      hivCD4Result_UUID: '657AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      hivCD4Count_UUID: '5497AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      hivDeathDate_UUID: '1543AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      causeOFDeath_UUID: 'ef973f1f-557f-4620-acf5-9c4c18bf1eda',
+      deathSpecific_UUID: 'e329cdf4-4eeb-4821-85ec-80ec4b503de0',
+      visitDate_UUID: '163137AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      receivingFacility_UUID: '162724AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      TransferOutDate_UUID: '160649AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      verified_UUID: '797e0073-1f3f-46b1-8b1a-8cdad134d2b3',
+      ContactDate_UUID: '160753AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      ContactMethod_UUID: '59c023dd-eed2-4b11-8c34-b88e9439db3c',
+      ContactOutcome_UUID: 'bc45edbd-11e7-4888-ad7d-4ec3dd8cdcf6',
+      PhysicalAbuse_UUID: '2a228c6a-1575-43d7-9d42-9b68d0629f46',
+      EmotionalAbuse_UUID: 'bd86f7ee-1d5f-4f5d-aa0f-4680aa6e65cb',
+      SexualAbuse_UUID: '1246AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      IpvScreeningDate_UUID: '160753AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      ContactTracingDate_UUID: '160753AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      ContactTracingMethod_UUID: '59c023dd-eed2-4b11-8c34-b88e9439db3c',
+      ContactTracingOutcome_UUID: '36a3e671-9d60-4109-b41f-046f44f4b389',
+      CommunityDSDModel_UUID: '52824cbe-0e4d-4c18-8179-80b5799f34ed',
+      EnrollmentDate_UUID: '160753AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      DSDStatus_UUID: '8742967d-8107-4cbb-a17e-9a8c7f624673',
+      DisclosureDate_UUID: '85fbdcc8-8dbc-40a9-b85f-5d1bfe1ab63d',
+      DisclosureStage_UUID: '573f93e2-12f6-483e-aa6e-14e9b76b311a',
+      IndexHIVStatus_UUID: '1436AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      PatnerNotificationContactDate_UUID: '160753AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      FirstName_UUID: '166102AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      Relationship_UUID: '85d3b4fe-c1a9-4e27-a86b-dcc1e30c8a93',
+      LittleInterestConcept_UUID: '167006AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      DepressionConcept_UUID: '167007AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      PoorAppetiteConcept_UUID: '167070AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      PoorConcentrationConcept_UUID: '167072AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      screeningDate_UUID: '160753AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      Cd4LabResultDate_UUID: '163724AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      Cd4LabResultCountPercentage_UUID: '730AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      Cd4Count_UUID: '5497AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      ViralLoadResultDate_UUID: '163724AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      ReasonForViralLoad_UUID: '86cc0cfe-bace-4969-94b6-d139f4971d13',
+      ViralLoadResult_UUID: '1305AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      ViralLoadCopies_UUID: '856AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      enrolmentDate: '160555AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
```

Comment:
@pirupius  I think the duplications is on the values and not on the keys so I did not look at that. And these keys are used in different places. I would suggest we have a follow up task to correct this issues otherwise we will compromise the work that this ticket is blocking. Agree?